### DISCUSSION TITLE
How to hide the left sidebar permanently?

### DISCUSSION THREAD

**Discussion by Riemmmannnn** [zed-industries/zed#11282] [Feature Requests]
Hello,

Is there any way to hide the left sidebar permanently? Whenever I restart, the left sidebar is opened automatically even I have closed it last time. I'd like to hide it permanently to keep a clean view. How can I achieve this?

Thanks.

**Comment by Moshyfawn** [comment]
This is a regression and will be fixed in future releases. Be sure to +👍 #11394.
